javascript/javascript2/week3/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Homework week3 js2</title>
+    
+</head>
+<body>
+    <section id="game-container">
+        <h1 id="game-headline">Fastest presser</h1>
+        <div id="input-button">
+    <input id="seconds-input"type="number" placeholder="set seconds">
+    <button id="start-btn">Start game!</button>
+    <br>
+    <p id="a-tie"></p>
+    <p id="countdown"></p>
+    </div>
+    <div id="letters">
+        <div>
+        <p id="press-l">Press l</p>
+    <p id="l"></p>
+    </div>
+        <p id="l-winner" data-text="l wins"></p>
+
+    <span id="line"></span>
+    <div>
+    <p>Press s</p>
+    <p id="s"></p>
+</div>
+    <p id="s-winner"></p>
+
+    </div>
+    </section>
+
+    <section id="warmup-container">
+    <div id="text-div">
+</div>
+<br>
+<button id="btn">Click here</button>
+<br>
+<br>
+<h3>Geolocation API Example:</h1>
```

Comment:
I would suggest having at least two different files for HTML and CSS, it makes it so much easier to get an overview of HTML and reduce the mistakes. For example, you are opening the wrong tag here, which is going to be an annoyance to find later on when you have so many lines of code!